### ISSUE TITLE
In Vue project, only vue files can be verified, other files (html,css,sass,scss,less files) don't verify

### ISSUE THREAD

**Issue by JowayYoung** [webpack-contrib/stylelint-webpack-plugin#224]
In Vue project, only vue files can be verified, other files (html,css,sass,scss,less files) don't verify. In React Project, as right as rain.

- Operating System: Windows 10
- Node Version: 14.14.0
- NPM Version: 6.14.8
- webpack Version: 4.44.2
- stylelint-webpack-plugin Version: 2.1.1

### Expected Behavior

In Vue project, Stylelint can verify html/css/sass/scss/less/vue files.

### Actual Behavior

In fact, only vue files can be verified, other files (html,css,sass,scss,less files) don't verify.

### Code

```js
new StylelintPlugin({
    cache: true,
    cacheLocation: AbsPath("node_modules/.cache/stylelint-webpack-plugin/.stylelintcache"),
    configFile: AbsPath(`../temp/configs/stylelintrc.${frame}.js`, 1),
    configOverrides: {
        ignoreFiles: stylelintIgnores.map(v => AbsPath(v)),
        rules: stylelintRules
    },
    context: process.cwd(),
    files: ["**/*.{html,css,sass,scss,less,vue}"]
})
```

**Comment by ricardogobbosouza** [comment]
Hi @JowayYoung 
Sorry for delay response.
Could you check if the problem still persists in version `v2.2.1` ?
If the problem persists could you provide a repository to reproduce?

**Comment by ricardogobbosouza** [comment]
Please reopen if the problem persists
